cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -81,6 +83,20 @@ data GenesisCreateStakedCmdArgs = GenesisCreateStakedCmdArgs
   , mStakePoolRelaySpecFile :: !(Maybe FilePath) -- ^ Relay specification filepath
   } deriving Show
 
+data GenesisCreateTestNetDataCmdArgs = GenesisCreateTestNetDataCmdArgs
+  { specShelley :: !(Maybe FilePath)
+  , numGenesisKeys :: !Word
+  , numPools :: !Word
+  , numStakeDelegators :: !Word
+  , numStuffedUtxo :: !Word
+  , numUtxoKeys :: !Word
+  , supply :: !(Maybe Lovelace)
```

Comment:
Maximum supply of lovelace right? A comment would be useful here too.

Reply:
It's the maximum supply of non-delegated lovelace, not the total supply. I propose to discuss the final API we want on Slack, and follow-up with another PR.